gui/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 2.8)
+project(OpenSpaceNet-gui)
+cmake_policy(SET CMP0015 NEW)
+# Find includes in corresponding build directories
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+# Instruct CMake to run uic automatically when needed
+set(CMAKE_AUTOUIC ON)
+# Instruct CMake to run moc automatically when needed
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_PREFIX_PATH $ENV{HOME}/Qt5.7.0/5.7/gcc_64)
+# Find the QtWidgets library
+find_package(Qt5Widgets REQUIRED)
+if (Qt5Widgets_FOUND)
+    message(Include Dirs: "${QT5WIDGETS_INCLUDE_DIRS}")
+endif()
+
+set(SOURCE_FILES
+        src/main.cpp
+        src/mainwindow.cpp
+        src/processthread.cpp
+        src/progresswindow.cpp
+        )
+
+set(HEADERS
+        include/mainwindow.h
+        include/processthread.h
+        include/progresswindow.h
+        include/qdebugstream.h
+        )
+
+add_executable(OpenSpaceNet-gui ${SOURCE_FILES} ${HEADERS})
+
+find_package(DeepCore REQUIRED)
+if (DeepCore_FOUND)
+    message(Include Dirs: "${DEEPCORE_INCLUDE_DIRS}")
+    include_directories(${DEEPCORE_INCLUDE_DIRS})
+    target_link_libraries(OpenSpaceNet-gui ${DEEPCORE_LIBRARIES})
```

Comment:
Also, the main reason I needed this.  You need the following to work with the tilejson part of DC.  (Joe has some ideas not to make this required.. but for now it is)
```
find_package(Jsoncpp REQUIRED)
if(JSONCPP_FOUND)
    include_directories(${JSONCPP_INCLUDE_DIRS})
    target_link_libraries(openspacenet ${JSONCPP_LIBRARIES})
endif()
```